akka-docs/src/main/paradox/io-tcp.md

```diff
@@ -1,3 +1,6 @@
+---
+project.description: Using TCP with Akka.
+---
```

Comment:
I don't think we should see this is a landing page, it's nowadays more like the low level underpinnings of other stuff, if people ask about doing TCP we'll very likely direct them to the stream tcp APIs instead. 

Maybe just not put a meta on this at all, or something like "Low level API for using TCP  with classic actors"?

Reply:
It seems users eventually want to know more about UDP/TCP options etc. so I thought it should get something for search-ability.